cpp/arcticdb/version/version_core.cpp

```diff
@@ -1905,7 +1943,7 @@ void set_row_id_if_index_only(
     if (read_query.columns &&
         read_query.columns->empty() &&
         pipeline_context.descriptor().index().type() == IndexDescriptor::Type::ROWCOUNT) {
-        frame.set_row_id(pipeline_context.rows_ - 1);
+        frame.set_row_id(static_cast<ssize_t>(pipeline_context.rows_ - 1));
```

Comment:
Not your problem, but confused why the `row_id` is a `ssize_t` but `pipeline_context.rows_` is `size_t`